testing/ember-app/tsconfig.json

```diff
@@ -9,7 +9,6 @@
     "paths": {
       "ember-app/tests/*": ["tests/*"],
       "ember-app/*": ["app/*"],
-      "ember-resources/*": ["../../ember-resources/dist/*"],
```

Comment:
@simonihmig, idk if this is relevant to anything you're working on, but I found that with dependenciesMeta*injected true that tsconfig's project references end up duplicating type declarations which ends up breaking type checking that relies on unique symbol types.